--- origsrc/gejengel-0.1.4/src/Plugins/notificationplugin.cpp	2010-03-19 09:31:43.000000000 -0500
+++ src/gejengel-0.1.4/src/Plugins/notificationplugin.cpp	2011-08-08 15:04:56.824351300 -0500
@@ -85,7 +85,7 @@ bool NotificationPlugin::initialize(Geje
         }
     }
 
-    m_pNotification = notify_notification_new("dummy", "", NULL, NULL);
+    m_pNotification = notify_notification_new("dummy", "", NULL);
     notify_notification_set_timeout (m_pNotification, NOTIFY_EXPIRES_DEFAULT);
     notify_notification_set_urgency (m_pNotification, NOTIFY_URGENCY_NORMAL);
 
